packages/design-system/src/__generated__/figma-design-tokens.ts

```diff
@@ -75,6 +78,9 @@ export const typography = {
     fontSize: "12px",
     lineHeight: "16px",
     letterSpacing: "0.005em",
+    textTransform: "capitalize",
```

Comment:
Not sure this will work well, as `text-transform: capitalize` might be unreliable. But I think we should try and see, before doing something more complicated